source/Mlos.Python/mlos/Optimizers/RegressionModels/RegressionEnhancedRandomForestModel.py

```diff
@@ -376,7 +299,7 @@ def _execute_grid_search_for_random_forest_regressor_model(self, x_filtered_to_d
             'n_estimators': [10, 50, 100],
             'max_features': max_feature_param
         }
-
+        print(f'RERF::performing random forest grid search CV')

```

Comment:
self.logger.info(f"Performing Random Forest Grid Search CV")

Reply:
Changed in latest commit